src/Kinepolis/KinepolisParser.php

```diff
@@ -54,6 +58,10 @@ public function getParsedMovies(array $moviesToParse): array
                 $is3D = $dimension === '3D';
                 $title = $is3D ? $title . ' 3D' : $title;
 
+                $isLong = $length >= 135;
+
+                $parsedPrice = $parsedPrices[$theatreId];
+
                 $calendar = sizeof($subEvents) === 1 ?
```

Comment:
count() is the more used method (3 times as much in the UDB3 code base), I would use that as a standard.

Reply:
Let us also create a follow-up ticket to remove all occurrences of `sizeof` and add a code style rule to always use `count` otherwise this change is meaningless.

Reply:
https://jira.publiq.be/browse/III-6150